### PR TITLE
Create bastion using ASG.

### PR DESCRIPTION
**Pull request by SiahaanBernard**
Update bastion module to create ec2 using Autoscaling group

### CODE REVIEW

File: examples/tsibstn/output.tf [hunk 1]
```diff
@@ -1,3 +1,24 @@
-output "instance_id" {
-  value = "${module.this.instance_id}"
+output "asg_bastion_name" {
+  description = "The name of the auto scaling group for bastion"
+  value       = "${module.this.asg_bastion_name}"
+}
+
+output "sg_bastion_id" {
+  description = "The name of the security group for bastion instance"
```
Comment (febryantonius) [inline examples/tsibstn/output.tf]: I think the description here should be updated to the appropriate one

**Comment by ayusun** [comment]
Hi 
@SiahaanBernard This seems to be perfect for ECI. We are looking to create Bastion to run our sqitch in this sprint. Is there any ETA when we can start using it?

**Comment by ayusun** [comment]
One more small question, is sqitch client baked into AMI along with ssm-agent?

**Comment by SiahaanBernard** [comment]
> One more small question, is sqitch client baked into AMI along with ssm-agent?

**Pull request merged**